resources/lang/en/metatags.php

```diff
@@ -55,7 +55,13 @@
     'exchanges'  => [
         'title'       => 'Exchanges and Aggregators | :name Blockchain Explorer',
         'description' => 'A list of active exchanges and aggregators/swaps that support ARK Token.',
-        'image'       => asset('images/metadata/wallets.png'), // TODO: update image
+        'image'       => asset('images/metadata/exchanges.png'),
+    ],
+
+    'wallets'  => [
```

Comment:
It might be worth renaming this to `compatible-wallets` to save confusion in the future